tests/conftest.py

```diff
@@ -401,7 +407,14 @@ def factory(
         else:
             secret = secret or secret_factory(interface=interface)
             if interface == constants.CEPHBLOCKPOOL:
-                interface_name = helpers.default_ceph_block_pool()
+                if config.ENV_DATA.get('new_rbd_pool') or new_rbd_pool:
+                    pool_obj = ceph_pool_factory(interface=interface,
+                                                 replica=config.ENV_DATA.get('replica') or replica,
+                                                 compression=config.ENV_DATA.get('compression') or compression
```

Comment:
Is it appropriate to provide an option to override  'replica' and 'compression' value in ENV_DATA?

```
                                                 replica=replica or config.ENV_DATA.get('replica'),
                                                 compression=compression or config.ENV_DATA.get('compression')
```

Reply:
We have planned to create new config files for creating new SC with replica and compression options , that is under WIP. https://github.com/red-hat-storage/ocs-ci/issues/3162
Checking for new pool creation , fetching compression and replica values from either from user end or from the configuration file, So I kept it under ENV_DATA